tests/fixtures/env/apps/harbor.yaml

```diff
@@ -8,6 +8,14 @@ apps:
                     rootdirectory: /google/demo
                 type: gcs
         core: {}
-        jobservice: {}
+        jobservice:
+            storageClassName: fast
         registry:
             credentials: {}
+            storageClassName: fast
+        # redis:
+        #     storageClassName: fast
```

Comment:
For testing purposes, I set a `storageClassName` for `redis` and `trivy`. It fails to sync. The message is
```
one or more objects failed to apply, reason: StatefulSet.apps "harbor-redis" is invalid: spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'ordinals', 'template', 'updateStrategy', 'persistentVolumeClaimRetentionPolicy' and 'minReadySeconds' are forbidden,StatefulSet.apps "harbor-trivy" is invalid: spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'ordinals', 'template', 'updateStrategy', 'persistentVolumeClaimRetentionPolicy' and 'minReadySeconds' are forbidden (retried 5 times).
```

Reply:
The `storageClassName` values have been removed and can not be set anymore.